client/components/ReviewRating/ReviewList/ReviewRating.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
```

Comment:
Let's try to be consistent with our imports
* external 
* components
* css
`import React, { useEffect, useState } from 'react';
import PropTypes from 'prop-types';
import axios from 'axios';

import ReviewList from './ReviewList';
import WriteReview from '../WriteReview/WriteReview';
import Sorting from '../Sorting/Sorting';

import '../../../css/reviewRating.scss';`